src/dattobd.c

```diff
@@ -737,10 +738,19 @@ static inline void dattobd_bio_copy_dev(struct bio *dst, struct bio *src){
 #define BIO_SET_SIZE 256
 #define bio_last_sector(bio) (bio_sector(bio) + (bio_size(bio) / SECTOR_SIZE))
 
-// As of Linux 5.2, __REQ_NR_BITS == 26
-#define __DATTOBD_PASSTHROUGH 30    /* don't perform COW operation */
+/* don't perform COW operation */
+#if defined HAVE_SUBMIT_BIO_1 && defined HAVE_ENUM_REQ_OP && !defined HAVE_ENUM_REQ_OPF
```

Comment:
Is the idea `defined HAVE_ENUM_REQ_OP` implies <= 4.9, and `defined HAVE_ENUM_REQ_OPF` implies >= 4.10? And if so, why `HAVE_SUBMIT_BIO_1` as well? Trying to understanding what each symbol means

Reply:
After sleeping on it, `HAVE_ENUM_REQ_OP` is the only required one, since the enum of that name only exists when the problem happens. The other flags were about me trying to mimic the definition cases for `dattobd_bio_op_set_flag` and others further up and are unnecessary.